Dockerfile

```diff
@@ -0,0 +1,35 @@
+# python 3
+FROM continuumio/anaconda3
+
+MAINTAINER Attawit Kittikrairit
+
+# set working directory
+WORKDIR /usr/src/app
+
+# clone from repo
+RUN git clone https://github.com/e-mission/e-mission-server.git .
+
+# setup python environment
+RUN conda env update --name emission --file setup/environment36.yml
+RUN /bin/bash -c "source activate emission; pip install six --upgrade"
+
+# install nodejs, npm and bower
+RUN apt-get update
+RUN apt-get install -y build-essential
```

Comment:
why do you need `build-essential`?
That installs the entire build toolchain including gcc.
We are not really building any c programs - `apt-get install nodejs` should install the compiled version of node...

Reply:
There are a lot of missing dependencies. many of npm packages requires compile from source when you do `npm install`.